ffda-gluon-ssh-manager/luasrc/lib/gluon/upgrade/860-ffda-gluon-ssh-manager

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/lua
+
+local uci = require('simple-uci').cursor()
+
+local dropbear_key_path = '/etc/dropbear/authorized_keys'
+local ssh_manager_block_begin = '#SSH-Manager-Group-Keys-Begin'
+
+local enabled = uci:get_bool('gluon-ssh-manager', 'settings', 'enabled')
+local groups = uci:get('gluon-ssh-manager', 'settings', 'groups')
+
+local function file_exists(file)
+	local f = io.open(file)
+	if not f then
+		return false
+	end
+	f:close()
+	return true
+end
+
+-- Abort if disabled or no group set
+if not enabled then
```

Comment:
removing the keys after being disabled sounds like something that should be done. the condition could be the presence of the block begin tag.